tests/flow/test_pause_replication_race.py

```
@@ -0,0 +1,96 @@
+PAUSE_MS            = 10   # duration of each CLIENT PAUSE pulse
+PAUSE_GAP_SEC       = 0.01 # gap between pulses, so postponed writes get a chance to drain
+
+stop_event = threading.Event()
+
+
+def _writer(env):
+    conn = env.getConnection()
+    while not stop_event.is_set():
+        try:
+            conn.execute_command("GRAPH.QUERY", GRAPH_ID, "CREATE (n:P) RETURN 1")
+        except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)